org-code-javabuilder/protocol/src/main/java/org/code/protocol/StatusMessageKey.java

```diff
@@ -9,5 +9,7 @@
   SENDING_VIDEO,
   TIMEOUT_WARNING,
   TIMEOUT,
-  EXITED
+  EXITED,
+  RUNNING_VALIDATION,
+  RUNNING_PROJECT_TESTS
```

Comment:
small nit: seems like these are grouped in rough order of execution, could we move these alongside RUNNING?

Reply:
done!